seed_message_sender/settings.py

```diff
@@ -255,3 +255,20 @@
 
 METRICS_URL = os.environ.get("METRICS_URL", None)
 METRICS_AUTH_TOKEN = os.environ.get("METRICS_AUTH_TOKEN", "REPLACEME")
+
+REDIS_HOST = os.environ.get("REDIS_HOST", "localhost")
+REDIS_PORT = os.environ.get("REDIS_PORT", 6379)
+REDIS_DB = os.environ.get("REDIS_DB", 0)
+
+CONCURRENT_VOICE_LIMIT = 0
+VOICE_MESSAGE_DELAY = 0  # seconds
+CONCURRENT_TEXT_LIMIT = 0
+TEXT_MESSAGE_DELAY = 0  # seconds
+
+CACHES = {
+    'default': {
+        'BACKEND': 'redis_cache.RedisCache',
+        'LOCATION': ['%s:%s' % (os.environ.get("REDIS_HOST", "localhost"),
+                                os.environ.get("REDIS_PORT", 6379))],
+    },
```

Comment:
I think we need the `OPTIONS` field here in order to use the `DB` value. We also might need the `PASSWORD` value.